parse.go

```diff
@@ -54,9 +54,10 @@ type spec struct {
 	separate      bool                // if true, each slice and map entry will have its own --flag
 	help          string              // the help text for this option
 	env           string              // the name of the environment variable for this option, or empty for none
-	defaultValue  reflect.Value       // default value for this option
-	defaultString string              // default value for this option, in string form to be displayed in help text
-	placeholder   string              // name of the data in help
+	envOnly       bool
```

Comment:
I think better to check `if spec.long == "" && spec.short == ""` rather than adding this variable to this struct

Reply:
:heavy_check_mark: